archive-engagement.php

```diff
@@ -0,0 +1,29 @@
+<?php session_start(); ?>
+
+<?php get_header(); ?>
+
+	<div class="project-grid accordion" id="accordionExample">
```

Comment:
Can you fix the indenting in this file and reduce the amount of empty lines between lines of code please.